packages/ui/src/uiManager.ts

```diff
@@ -57,4 +61,46 @@ export class UIManager {
         this._options.bannerManager && this._options.bannerManager.reset();
         this._options.toastManager && this._options.toastManager.reset();
     }
+
+    private _loadFont(): void {
+        let fontFamily = this._options.corePlayer.config.contrib.ui.fonts.fontFamily;
+
+        const fontCss = `.kaltura-player-container {
+                font-family: inherit;
+            }         
+            .playkit-player {
+                font-family: ${fontFamily};
+            }
+            .playkit-player .playkit-player-gui  {
+                font-family: ${fontFamily};
+            }
+            button, textarea {
+                font-family: inherit;
+            }`;
+        const lastHeadChild = (document.head || document.getElementsByTagName("head")[0])
+            .lastElementChild;
+        const style = document.createElement("style");
+        //adding as last child
+        lastHeadChild.parentNode.insertBefore(style, lastHeadChild.nextSibling);
```

Comment:
shouldn't it be `insertAfter` to be the last one?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Node/insertBefore
```
There is no insertAfter() method. It can be emulated by combining the insertBefore method with Node.nextSibling.
```